### PR TITLE
linking requirements.txt in .pyproject.toml

### LINKED ISSUE #8: update pip install dependencies
The list of dependencies from the `.pyproject.toml` file is outdated, this doesn't install new dependencies such as `dask` and `xarray`.

To reproduce, clone the current project,. then execute de following: 
```
pip install -e .

python -c "import torch; print(torch.__version__)"
2.4.0+cu121

python -c "import dask; print(dask.__version__)"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'dask'
```

### PR DESCRIPTION
**Pull request by jfbourgon**
closes #8

### CODE REVIEW

**Review by mpelchat04 — APPROVED** [review]

File: VERSION [hunk 1]
```diff
@@ -0,0 +1 @@
+2.0.7
```
Comment (valhassan) [inline VERSION:1]: are you using a different tool to create the VERSION file?

File: pyproject.toml [hunk 2]
```diff
@@ -4,7 +4,6 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "geo-inference"
-version = "2.0.7"
```
Comment (valhassan) [inline pyproject.toml:7]: This was updated automatically using bumpver,

File: VERSION (hunk 1 above)
Comment (jfbourgon) [reply VERSION:1]: I added the VERSON in the list of file managed by bumpver

File: pyproject.toml (hunk 2 above)
Comment (jfbourgon) [reply pyproject.toml:7]: Yes it is, here is an updated output of the bumpver command with the new conifg

```
bumpver update --minor --dry
INFO    - fetching tags from remote (to turn off use: -n / --no-fetch)
INFO    - Old Version: 2.0.7
INFO    - New Version: 2.1.0
--- VERSION
+++ VERSION
@@ -1 +1 @@
-2.0.7
+2.1.0
--- geo_inference/__init__.py
+++ geo_inference/__init__.py
@@ -1,5 +1,5 @@
 """Geo-inference: Extract features from high-resolution geospatial imagery using foundation models"""

 __author__ = "Victor Alhassan"
-__version__ = "2.0.7"
+__version__ = "2.1.0"

--- pyproject.toml
+++ pyproject.toml
@@ -37,7 +37,7 @@
 include = ["geo_inference*"]

 [tool.bumpver]
-current_version = "2.0.7"
+current_version = "2.1.0"
 version_pattern = "MAJOR.MINOR.PATCH"
 commit_message = "bump version {old_version} -> {new_version}"
 commit = true

```

**Review by valhassan — APPROVED** [review]

**Pull request merged**